README.md

```diff
@@ -127,22 +199,87 @@ Failure to consume an error will result in the same unconsumed event exception a
 with the exception added as the cause so that the full stacktrace is available.
 
 ```kotlin
-flow { throw RuntimeException("broken!") }.test { }
+flow<Nothing> { throw RuntimeException("broken!") }.test { }
 ```
 ```
-java.lang.AssertionError: Unconsumed events found:
+app.cash.turbine.TurbineAssertionError: Unconsumed events found:
  - Error(RuntimeException)
-    at app.cash.turbine.ChannelBasedFlowTurbine.ensureAllEventsConsumed(FlowTurbine.kt:240)
-    ... 53 more
+	at app//app.cash.turbine.ChannelTurbine.ensureAllEventsConsumed(Turbine.kt:215)
+  ... 80 more
 Caused by: java.lang.RuntimeException: broken!
-    at example.MainKt$main$1.invokeSuspend(Main.kt:7)
-    ... 32 more
+	at example.MainKt$main$1.invokeSuspend(FlowTest.kt:652)
+	... 105 more
+```
+
+While flow termination is exposed as an event, it is not really an event: it's a state that the flow is in.
+So completion or error events can be observed multiple times.
+
+```kotlin
+flow<Nothing> { throw RuntimeException("broken!") }.test {
+  awaitError()
+  awaitError()
+  awaitError()
+}
 ```
```

Comment:
Not sure whether this meets that bar for the intro/README. It's a great snippet to put on the kdoc of each of these functions (which we should do no matter what), but I'm unsure as to whether it's something important enough to actively communicate here. What do you think?

Reply:
This is already documented well enough, I think:

```
  /**
   * Assert that the next event received was an error terminating the flow.
   * This function will suspend if no events have been received.
   *
   * When this [ReceiveTurbine] is in a terminal state ([Event.Complete] or [Event.Error], this method
   * will yield the same result every time it is called.
   *
   * @throws AssertionError if the next event was an item or completion.
   */
```